roles/openshift-applier/tasks/process-one-entry.yml

```diff
@@ -19,119 +20,20 @@
   when:
   - not provision|bool
 
-- name: "Fail if template but no params is specified"
-  fail:
-    msg: "Template specified, but no params file supplied"
-  when:
-  - template|trim != ''
-  - params|trim == ''
-
-- name: "Check if params file exists (if applicable)"
-  stat:
-    path: "{{ tmp_inv_dir }}{{ params }}"
-  ignore_errors: yes
-  register: params_result
-  when:
-  - params|trim != ''
-
-- name: "Fail if params file doesn't exist (if applicable)"
-  fail:
-    msg: "{{ params }} - params file doesn't exist."
-  when:
-  - params|trim != ''
-  - params_result.stat.exists == False
-
-- name: "Determine location for the 'file' (if applicable)"
-  vars:
-    path: "{{ file }}"
-  import_tasks: check-file-location.yml
-  when:
-  - file|trim != ''
-
-- name: "Set 'file' values (if applicable)"
-  set_fact:
-    file: "{{ file_path }}"
-    file_f_option: "{{ option_f }}"
-  when:
-  - file|trim != ''
-
-- name: "Determine location for the 'template' (if applicable)"
-  vars:
-    path: "{{ template }}"
-  import_tasks: check-file-location.yml
-  when:
-  - template|trim != ''
-
-- name: "Set 'template' values (if applicable)"
-  set_fact:
-    template: "{{ file_path }}"
-    template_f_option: "{{ option_f }}"
-  when:
-  - template|trim != ''
-
 - name: "Set the target namespace option if supplied"
   set_fact:
     target_namespace: "-n {{ content.namespace }}"
   when:
   - content.namespace is defined
   - content.namespace|trim != ''
 
-- name: "Include any pre-processing role(s) before applying file and/or template"
```

Comment:
While splitting the file and template processing out into separate files may be ok, it looks like the pre/post processing should be kept here to avoid duplicates in those separate files.

Reply:
that's fine with me. I thought there was some issue with keeping them here, but i can't remember now. I'll move them back.